src/riscv/lib/src/machine_state/block_cache/state.rs

```diff
@@ -17,7 +17,6 @@ use crate::machine_state::instruction::Instruction;
 use crate::machine_state::memory::Address;
 use crate::machine_state::memory::MemoryConfig;
 use crate::machine_state::memory::OFFSET_MASK;
```

Comment:
If this import still exists, I think there might still be a page boundary check.